clang/test/CIR/hello.c

```diff
@@ -0,0 +1,4 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s
```

Comment:
Would still like this to 'pipe' to filecheck.  In addition to 'not crash', it would be worthwhile to ensure we emit sensible IR.

Reply:
We don't have any IR to test just yet. @lanza you could use `FileCheck` with `--allow-empty` here for now. You could also rename this test `emit-cir.c` since the purpose for now is to not crash (as it would before this PR).

Reply:
> We don't have any IR to test just yet. @lanza you could use `FileCheck` with `--allow-empty` here for now. You could also rename this test `emit-cir.c` since the purpose for now is to not crash (as it would before this PR).

Both of those are OK enough ideas to me.  So at this point, we're just emitting a blank-file, right?   Perhaps   `--allow-empty` plus a `// CHECK-NOT: *` or something.